### PR TITLE
Update the 0904meetup and fix the typo of 0828meetup

### PR DESCRIPTION
**Pull request by ThreeMonth03**
According to the issue #458, I update the information of meetup in 09/04. BTW, I also fix the minor typo in the webpage of 08/28 meetup.

### CODE REVIEW

File: content/pages/meetup/2024/0904-nycu.rst [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+* Task arrangement for volunteers
+* Promotion plan for upcoming events
+
+|
+
+Sign up
+------------
+
+The meetup is free. 
+Please register on `discord event <https://discord.com/channels/730297880140578906/1007075707400237067/1275445577521565837>`__.
```
Comment (yungyuc) [inline content/pages/meetup/2024/0904-nycu.rst:79]: This links to 8/21 event.  Please correct.  (Maybe you need to create the event before making this PR.)

image

**Review by yungyuc — CHANGES_REQUESTED** [review]
@ThreeMonth03 Some corrections are needed.  @kuomc @YenPeiChen07 @Gene0315, please take a look.

File: content/pages/meetup/2024/0828-nycu.rst [hunk 2]
```diff
@@ -76,7 +76,7 @@ Sign up
 ------------
 
 The meetup is free. 
-Please register on `discord event <https://discord.com/channels/730297880140578906/1007075707400237067/1275445577521565837>`__. 
+Please register on `discord event <https://discord.com/channels/730297880140578906/1007075707400237067/1277532036042719262>`__.
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/0828-nycu.rst:79]: fixed

File: content/pages/meetup/2024/0904-nycu.rst [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+Sign up
+------------
+
+The meetup is free. 
+Please register on `discord event <https://discord.com/channels/730297880140578906/1007075707400237067/1279083158738042996>`__. 
+Click the green check mark to participate the meetup.
+
+If you are using the discord app, you can find current event in the `meetup channel <https://discordapp.com/channels/730297880140578906/1007075707400237067>`__. 
+All recent sciwork event are at the top of the left sidebar.
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/0904-nycu.rst:87]: fixed

File: content/pages/meetup/2024/0904-nycu.rst [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
++-----------------+--------------------------+-------------------------------+------------------------------------+
+| Timetable       | Topic 1                  | Topic 2                       | Topic 3                            |
++=================+==========================+===============================+====================================+
+| 18:30 ~ 19:30   | Free Chat                                                                                     |
++-----------------+--------------------------+-------------------------------+------------------------------------+
+| 19:30 ~ 21:00   | Free Chat                | Modmesh Basic Training        | Modmesh Development Discussion     |
++-----------------+--------------------------+-------------------------------+------------------------------------+
+| 21:00 ~ 21:30   | Free Chat                                                                                     |
++-----------------+--------------------------+-------------------------------+------------------------------------+
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/0904-nycu.rst:30]: timetable

Comment (ThreeMonth03) [reply content/pages/meetup/2024/0904-nycu.rst:30]: @visitorckw will not attend the meetup next week.

**Review by ThreeMonth03 — COMMENTED** [review]
- [x] Fix the error link.
 - [x] Add the clear timetable and topic discription.

**Comment by ThreeMonth03** [comment]
> * [x]  Fix the error link.
> * [x]  Add the clear timetable and topic discription.

@yungyuc @kuomc, you could review this pr if you are available. Thank you.

File: content/pages/meetup/2024/0904-nycu.rst (hunk 4 above)
Comment (yungyuc) [reply content/pages/meetup/2024/0904-nycu.rst:30]: image

It looks pretty nice!

@visitorckw, could you please let the team know when you plan to host a topic in the meetup?

**Review by yungyuc — APPROVED** [review]
The table looks nice

image

@ThreeMonth03, do you know who will lead the discussion for the modmesh development?  Maybe @Gene0315 would help?

**Comment by ThreeMonth03** [comment]
> The table looks nice
> 
> image
> @ThreeMonth03, do you know who will lead the discussion for the modmesh development? Maybe @Gene0315 would help?

https://github.com/sciwork/swportal/blob/fdd5ea33d2ef01f5c9e6e66d9e1e8276a233d767/content/pages/meetup/2024/0904-nycu.rst?plain=1#L38-L49

In the following topic description, I mentioned that I will lead the discussion of [issue343](https://github.com/solvcon/modmesh/issues/343) for those who are interested in modmesh, and they also don't need the basic training. I'm not sure whether @Gene0315 is willing to lead the advanced discussion.

**Comment by yungyuc** [comment]
> > @ThreeMonth03, do you know who will lead the discussion for the modmesh development? Maybe @Gene0315 would help?
> 
> https://github.com/sciwork/swportal/blob/fdd5ea33d2ef01f5c9e6e66d9e1e8276a233d767/content/pages/meetup/2024/0904-nycu.rst?plain=1#L38-L49
> 
> @yungyuc In the following topic description, I mentioned that I will lead the discussion of [issue343](https://github.com/solvcon/modmesh/issues/343) for those who are interested in modmesh, and they also don't need the basic training. I'm not sure whether @Gene0315 is willing to lead the advanced discussion.

Thanks.  I think the contents of the program is good.  If @Gene0315 does not notice the conversation in this PR, could you please follow up with her?

**Comment by ThreeMonth03** [comment]
> > > @ThreeMonth03, do you know who will lead the discussion for the modmesh development? Maybe @Gene0315 would help?
> > 
> > 
> > https://github.com/sciwork/swportal/blob/fdd5ea33d2ef01f5c9e6e66d9e1e8276a233d767/content/pages/meetup/2024/0904-nycu.rst?plain=1#L38-L49
> > 
> > @yungyuc In the following topic description, I mentioned that I will lead the discussion of [issue343](https://github.com/solvcon/modmesh/issues/343) for those who are interested in modmesh, and they also don't need the basic training. I'm not sure whether @Gene0315 is willing to lead the advanced discussion.
> 
> Thanks. I think the contents of the program is good. If @Gene0315 does not notice the conversation in this PR, could you please follow up with her?

Yes, I later discussed the meetup topic with her privately, and we decided to go over the details this Wednesday.

**Review by kuomc — APPROVED** [review]

**Comment by yungyuc** [comment]
Thanks.  @ThreeMonth03 , you have got familiar with the updating and publishing flow.  I added you into to repository maintainer.  You will be able to merge PR in the future.

**Pull request merged**